src/glib/base/ds.hpp

```diff
@@ -381,9 +381,9 @@ void TVec<TVal, TSizeTy>::GetSubValV(const TSizeTy& _BValN, const TSizeTy& _EVal
   const TSizeTy BValN=TInt::GetInRng(_BValN, 0, Len()-1);
   const TSizeTy EValN=TInt::GetInRng(_EValN, 0, Len()-1);
   const TSizeTy SubVals=TInt::GetMx(0, EValN-BValN+1);
-  SubValV.Gen(SubVals, 0);
-  for (TSizeTy ValN=BValN; ValN<=EValN; ValN++){
-    SubValV.Add(GetVal(ValN));}
+  SubValV.Resize(SubVals);
+  memcpy(SubValV.ValT, ValT + BValN, SubVals * sizeof(TVal));
```

Comment:
We have few other instances of this in TVec, and so far usually created a separate function to cover use-case where `memcpy` is a valid option (e.g. TIntV or TFltV).